crates/cli/src/commands/install.rs

```diff
@@ -55,17 +57,19 @@ impl PackageManager {
         package
             .dependencies(self.config.auto_install_peers)
             .map(|(name, version)| async {
+                let semaphore_clone = semaphore.clone();
```

Comment:
`semaphore` is not an `Arc` here. Cloning it would create a new `Semaphore`. Is this intentional?